services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -195,6 +257,8 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
     lazy val withRepoProject: Project =
       activeProject.withRepository(Some(s"${gitLabConfig.idPath}${activeProject.projectId.value}"))
     lazy val dummyVersion: Version = TestProjectUtils.getDummyVersion()
+    lazy val dummyFileCommit: FileCommit = TestProjectUtils.getDummyFileCommit()
+    lazy val dummyFileCommitAux: FileCommit = FileCommit(dummyVersion.commit.id)
```

Comment:
Use just TestProjectUtils.getDummyFileCommit(). It will be different file commit

Reply:
Same, the process needs an existing commit